### PR TITLE
fix: geographic address handling

### PR DESCRIPTION
**Pull request by elysee15**
Capture d’écran 2021-09-28 à 19 20 24

Fixes SC-610

### CODE REVIEW

File: web/gds-admin-ui/src/pages/app/details/ivms/Geographic.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    }
+
+
+    return (
+        <Col lg={6} className="mt-3">
+            <p className="fw-bold mb-2">Geographic Adressess</p>
+            <hr />
+            {
+                data && data.map((address, index) => (
+                    <Row key={index}>
```
Comment (bbengfort) [inline web/gds-admin-ui/src/pages/app/details/ivms/Geographic.js:75]: Let's not forget to render the address type header also!

**Review by bbengfort — COMMENTED** [review]
@elysee15 phenomenal work! I really like the decomposition of the various IVMS 101 elements into their own components; this will help us organize the detail page even better. And the addresses are readable now! 

I had a few comments, if you wouldn't mind addressing them.

File: web/gds-admin-ui/src/pages/app/details/contact/AdministrativeContact.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                    <p className="fw-bold mb-2">Email: <span className="fw-normal">{formatDisplayedData(data?.email)}</span></p>
+                    <p className="fw-bold mb-2">Name: <span className="fw-normal">{formatDisplayedData(data?.name)}</span></p>
+                </Col>
+                <Col xl={6}>
+                    <p className="fw-bold mb-2">Person: <span className="fw-normal">{formatDisplayedData(data?.person)}</span></p>
+                    <p className="fw-bold mb-2">Email: <span className="fw-normal">{formatDisplayedData(data?.phone)}</span></p>
+                </Col>
+            </Row>
+        </>
+    )
```
[the remark this answers is not in the document]
Comment (elysee15) [reply web/gds-admin-ui/src/pages/app/details/contact/AdministrativeContact.js:22]: Yes good idea

File: web/gds-admin-ui/src/pages/app/details/contact/Contact.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+
+import React from 'react'
+import { Col, Row } from 'react-bootstrap';
+import { formatDisplayedData } from '../../../../utils';
+
+function Contact({ data, type }) {
+    return (
+        <>
+            <p className="fw-bold">{type} contact :</p>
```
Comment (bbengfort) [inline web/gds-admin-ui/src/pages/app/details/contact/Contact.js:9]: Minor typo (in English) - could we please remove the space between contact and :, e.g. `contact:</p>`?

File: web/gds-admin-ui/src/pages/app/details/contact/index.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Card, Col, Row } from 'react-bootstrap';
+import Contact from './Contact';
+
+export default function ContactList({ data }) {
+
+    return (
+        <Card>
+            <Card.Body>
+                <h4 className="mt-0 mb-3">Contact Informations</h4>
```
Comment (bbengfort) [inline web/gds-admin-ui/src/pages/app/details/contact/index.js:10]: Minor typo: "Information" not "Informations".

**Review by bbengfort — APPROVED** [review]
@elysee15 well done; it looks very good. I've marked a few minor typos - would you mind updating them in your next PR? I'm going to go ahead and get this merged.

**Pull request merged**